app/serializers/search_and_compare/course_serializer.rb

```diff
@@ -65,8 +69,32 @@ class CourseSerializer < ActiveModel::Serializer
     attribute(:FullTime)                              { object.part_time? ? 3 : 1 }
     attribute(:PartTime)                              { object.full_time? ? 3 : 1 }
 
+    # Course_direct_enrichment_Mapping
+    attribute(:Duration)                              { course_enrichment.duration }
```

Comment:
Can we move the duration logic in the method into here?

Reply:
Moved duration to serializer